### DISCUSSION TITLE
out of date AUR package

### DISCUSSION THREAD

**Discussion by matteoscaringi** [airsquared/blobsaver#340] [Q&A] [answered]
Please do not submit a bug report or feature request using this template, your issue will be closed and/or deleted.

Remove the following line and the text above it before submitting this issue:

-------------------------------------------------------------
Did you put the AUR package for arch on the repos? im just here to tell ya that it is out of date and the updater in the program does not work on linux.

**Comment by airsquared** [comment, marked as answer]
I didn't put the package on AUR myself, and I don't use Arch Linux so I can't update the package. I wouldn't recommend using the package on AUR unless someone would be willing to update it.

Edit: there's now someone who updates the package pretty frequently, so now I'd recommend using the AUR package

**Reply by matteoscaringi** [reply]
i already compiled it form source
